titus-common/src/test/java/com/netflix/titus/common/util/StringExtTest.java

```diff
@@ -81,4 +81,11 @@ public void testParseDurationList() {
                 Duration.ofMillis(1), Duration.ofMillis(2), Duration.ofMillis(3)
         );
     }
+
+    @Test
+    public void testGzipAndBase64Encode() {
+        String input = "{aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa:bbbbbbbbbbbbbbbbbbbbbbbbbbb}";
+        String output = StringExt.gzipAndBase64Encode(input);
+        assertThat(output.length()).isLessThan(input.length());
```

Comment:
Looks like this assertion is unique to the input in this case.
Should we generalize the test and possibly validate the decoded-gunzipped output is the same as input?
Also empty string is an interesting case since its output is non-empty. Maybe also handle that instead of returning an empty string. Curious to know what you think.

Reply:
I didn't go do that because I didn't write the decode and ungzip method, but I will think about this some more and potentially change it in a followup PR.